schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+
+CREATE TABLE patients (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE medical_histories (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    admitted_at TIMESTAMP,
+    patient_id INT,
+    status VARCHAR(50),
+    PRIMARY KEY (id),
+    FOREIGN KEY (patient_id) REFERENCES patiens(id)
+);
+
+CREATE TABLE invoices (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    total_amount REAL,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT,
+    PRIMARY KEY (id),
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
+);
```

Comment:
- [ ] Great job on your implementation so far! 👏 However, I'd like to point out a minor issue that, when corrected, will bring your project in line with the specified requirements.

    - 🔍 When examining the `ERD` (Entity-Relationship Diagram) provided in the project requirements, you'll notice that the column `total_amount` is defined with a data type of `DECIMAL`. However, in your code, you've defined `total_amount` with a data type of `REAL`, which is not in line with the requirements.

        - To ensure your project's accuracy and alignment with the requirements, please consider making this change to the data type of `total_amount` to match the `DECIMAL` type indicated in the `ERD`.

Happy coding! 😊🚀👨‍💻